OBAKit/Theme/Icons.swift

```diff
@@ -14,25 +14,44 @@ import OBAKitCore
 class Icons: NSObject {
 
     // MARK: - Tab Icons
-
     /// The Map tab icon, for apps using a tab bar UI metaphor.
     public class var mapTabIcon: UIImage {
-        imageNamed("map")
+        configureForTabIcon(systemImage(named: "map", fallback: "map"))
+    }
+
+    /// The Map tab selected icon, for apps using a tab bar UI metaphor.
+    public class var mapSelectedTabIcon: UIImage {
+        configureForTabIcon(systemImage(named: "map.fill", fallback: "map_selected"))
     }
 
     /// The Recent tab icon, for apps using a tab bar UI metaphor.
     public class var recentTabIcon: UIImage {
-        imageNamed("recent")
+        configureForTabIcon(systemImage(named: "clock", fallback: "recent"))
+    }
+
+    /// The Recent tab selected icon, for apps using a tab bar UI metaphor.
+    public class var recentSelectedTabIcon: UIImage {
+        configureForTabIcon(systemImage(named: "clock.fill", fallback: "recent_selected"))
     }
 
     /// The Bookmarks tab icon, for apps using a tab bar UI metaphor.
     public class var bookmarksTabIcon: UIImage {
-        imageNamed("bookmarks")
+        configureForTabIcon(systemImage(named: "bookmark", fallback: "bookmark"))
+    }
+
+    /// The Bookmarks tab selected icon, for apps using a tab bar UI metaphor.
+    public class var bookmarksSelectedTabIcon: UIImage {
+        configureForTabIcon(systemImage(named: "bookmark.fill", fallback: "bookmark_selected"))
     }
 
     /// A More tab icon, for apps using a tab bar UI metaphor.
     public class var moreTabIcon: UIImage {
-        imageNamed("more")
+        configureForTabIcon(systemImage(named: "ellipsis.circle", fallback: "more"))
```

Comment:
please change to `systemImage(named: "ellipsis", fallback: "more")` and drop the `moreSelectedTabIcon`

Reply:
This is a workaround for an alignment issue with `ellipsis`.
![Simulator Screen Shot - iPhone 11 Pro - 2020-07-27 at 23 21 13](https://user-images.githubusercontent.com/22162410/88626717-ec1cc380-d05f-11ea-9696-d954ea72c3ad.png)